packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -918,6 +926,8 @@ class DefaultTextStyleTransition extends AnimatedWidget {
 /// ```
 /// {@end-tool}
 ///
+/// ![A spinning container.](https://flutter.github.io/assets-for-api-docs/assets/widgets/aniamted_builder.mp4)
```

Comment:
In review here: https://github.com/flutter/assets-for-api-docs/pull/107

Reply:
This should use the `{animation...}` embedding format correct?

Reply:
Yes! Thanks for the catch!